resources/console/profile-production.yaml

```diff
@@ -22,4 +22,12 @@ backend:
       memory: 40Mi
     limits:
       cpu: 50m
-      memory: 150Mi
+      memory: 200Mi
```

Comment:
@dariadomagala I raised this value. I assumed the man with the problem was using a production cluster.

Reply:
as we discussed, this issue should be moved to a separate task after all, please remove this change :)